root/src/main/resources/templates/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
```

Comment:
Is this supposed to be the same file as in mymovielist/index.html? Seems like it's a bit different

Reply:
They are the same files. But I reformatted the html file and I guess prettier made some changes as well.

Reply:
It looks pretty different to me, here's what the latest version looks like: https://github.com/The-Movie-List-Makers/mymovielist/blob/main/index.html

It has a footer and uses a div and figure tags instead of an ordered list